docker/etc/pacman/pacman.latest.conf

```diff
@@ -0,0 +1,28 @@
+# Alternative pacman config file which downloads from the latest version of archlinux.
+# You probably *should not* use this file.
```

Comment:
So why are we using it? :thinking:

Reply:
The install-package script is using it to specify the `Server=` value. As far as I know there is on other way to tell pacman to pull from a different server value.